solr/test-framework/src/java/org/apache/solr/SolrTestCaseJ4.java

```diff
@@ -1849,73 +1914,90 @@ public Fld createField() {
       Fld fld = new Fld();
       fld.ftype = this;
       fld.vals = vals;
-      return fld;          
+      return fld;
     }
-
   }
 
   public static void assertResponseValues(SolrResponseBase rsp, Object... assertions) {
     Map<String, Object> values = Utils.makeMap(assertions);
-    values.forEach((s, o) -> {
-      if (o instanceof String) {
-        assertEquals(o, rsp.getResponse()._getStr(s, null));
-      } else {
-        assertEquals(o, rsp.getResponse()._get(s, null));
-      }
-    });
+    values.forEach(
+        (s, o) -> {
+          if (o instanceof String) {
+            assertEquals(o, rsp.getResponse()._getStr(s, null));
+          } else {
+            assertEquals(o, rsp.getResponse()._get(s, null));
+          }
+        });
   }
+
   @SuppressWarnings({"rawtypes"})
-  public Map<Comparable,Doc> indexDocs(List<FldType> descriptor, Map<Comparable,Doc> model, int nDocs) throws Exception {
+  public Map<Comparable, Doc> indexDocs(
+      List<FldType> descriptor, Map<Comparable, Doc> model, int nDocs) throws Exception {
     if (model == null) {
       model = new LinkedHashMap<>();
     }
 
     // commit an average of 10 times for large sets, or 10% of the time for small sets
-    int commitOneOutOf = Math.max(nDocs/10, 10);
+    int commitOneOutOf = Math.max(nDocs / 10, 10);
 
-    for (int i=0; i<nDocs; i++) {
+    for (int i = 0; i < nDocs; i++) {
       Doc doc = createDoc(descriptor);
       // doc.order = order++;
       updateJ(toJSON(doc), null);
       model.put(doc.id, doc);
 
       // commit 10% of the time
-      if (random().nextInt(commitOneOutOf)==0) {
+      if (random().nextInt(commitOneOutOf) == 0) {
         assertU(commit());
       }
 
       // duplicate 10% of the docs
-      if (random().nextInt(10)==0) {
+      if (random().nextInt(10) == 0) {
         updateJ(toJSON(doc), null);
-        model.put(doc.id, doc);        
+        model.put(doc.id, doc);
       }
     }
 
     // optimize 10% of the time
-    if (random().nextInt(10)==0) {
+    if (random().nextInt(10) == 0) {
       assertU(optimize());
     } else {
       if (random().nextInt(10) == 0) {
         assertU(commit());
       } else {
-        assertU(commit("softCommit","true"));
+        assertU(commit("softCommit", "true"));
       }
     }
 
-    // merging segments no longer selects just adjacent segments hence ids (doc.order) can be shuffled.
+    // merging segments no longer selects just adjacent segments hence ids (doc.order) can be
+    // shuffled.
     // we need to look at the index to determine the order.
-    String responseStr = h.query(req("q","*:*", "fl","id", "sort","_docid_ asc", "rows",Integer.toString(model.size()*2), "wt","json", "indent","true"));
+    String responseStr =
+        h.query(
+            req(
+                "q",
```

Comment:
ok for now but if I were writing fresh code and saw spotless do this, I would go about writing the code differently.  I'd call `params().set("q", "*:*").set(...).set(...)` etc.